coralproject/talk
# Flags locales

## What does this PR do?

- Flags now are all required to pre-define their types in the graph, or at the very least provide their translations in locales.

Ref: https://www.pivotaltracker.com/story/show/151179316

## How do I test this PR?

1. Create some flags
2. Check out the translated responses in the admin
